ps_landing_page/models/hr_employee_landing_page.py

```diff
@@ -183,11 +202,28 @@ def _default_employee(self):
     )
     current_week = fields.Boolean(compute="_compute_all")
 
+    def last_employement_week(self):
```

Comment:
this name is misleading, as you select the *first* employment week here

Reply:
@hbrunn, my bad I couldn't realize this mistake, will change this.